src/app/[username]/components/profile/ProfileCard.tsx

```diff
@@ -40,7 +46,7 @@ export const ProfileCard = ({
 
   return (
     <div className='space-y-2'>
-      <ProfileFrame className='size-28 sm:size-32'>
+      <ProfileFrame className='size-[100px] md:size-[120px]'>
```

Comment:
Create a JSON that contains schema for socials and respective urls, using that will get rid of the repitions here

Reply:
Will do changes on this later